illwill.nim

```diff
@@ -334,13 +465,27 @@ when defined(windows):
     discard writeConsole(hStdout, pointer(us[0].addr), DWORD(s.runeLen),
                          numWritten.addr, nil)
 
-
 else:  # OS X & Linux
   import posix, tables, termios
+  import strutils # TODO mouse
 
   proc consoleInit()
   proc consoleDeinit()
 
+  # Mouse
+  # https://de.wikipedia.org/wiki/ANSI-Escapesequenz
+  # https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Extended-coordinates
+  const
+    CSI = 0x1B.chr & 0x5B.chr
+    SET_BTN_EVENT_MOUSE = "1002"
+    SET_ANY_EVENT_MOUSE = "1003"
+    SET_SGR_EXT_MODE_MOUSE = "1006"
+    SET_URXVT_EXT_MODE_MOUSE = "1015"
+    ENABLE = "h"
+    DISABLE = "l"
+    mouseTrackAny = fmt"{CSI}?{SET_BTN_EVENT_MOUSE}{ENABLE}{CSI}?{SET_ANY_EVENT_MOUSE}{ENABLE}{CSI}?{SET_SGR_EXT_MODE_MOUSE}{ENABLE}"
+    disableMouseTrackAny = fmt"{CSI}?{SET_BTN_EVENT_MOUSE}{DISABLE}{CSI}?{SET_ANY_EVENT_MOUSE}{DISABLE}{CSI}?{SET_SGR_EXT_MODE_MOUSE}{DISABLE}"
```

Comment:
I'm not 100% consistent with this either but we should strive to use UpperCamelCase for constant names (the exception is when defining constants from C header files).